tests/unit/splinterdb_stress_test.c

```diff
@@ -219,60 +219,6 @@ CTEST2(splinterdb_stress, test_iterator_over_many_kvs)
    splinterdb_iterator_deinit(it);
 }
 
-/*
- * Test case that inserts large # of KV-pairs, and goes into a code path
- * reported by issue# 458, tripping a debug assert. This test case also
- * triggered the failure(s) reported by issue # 545.
- * FIXME: This test still runs into an assertion "filter->addr != 0"
- * from trunk_inc_filter(), which is being triaged separately.
- */
-CTEST2_SKIP(splinterdb_stress, test_issue_458_mini_destroy_unused_debug_assert)
```

Comment:
Test case relocated to `unit/large_inserts_stress_test.c`, as it better belongs to that test file.